### PR TITLE
moved cd/m2 to name column

### PR DESCRIPTION
**Pull request by digitaltvguy**
This moves cd/m2 from divisor column and moves it to the name column

### CODE REVIEW

**Review by svgeesus — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3904,13 +3904,13 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
             </tr>
 
             <tr>
-              <td>Mastering display maximum luminance</td>
+              <td>Mastering display maximum luminance (measured in cd/m<sup>2</sup>)</td>
               <td>4 bytes</td>
-              <td>0.0001 cd/m<sup>2</sup></td>
```
Comment (svgeesus) [inline index.html:3909]: why is this removed here, but not in min luminance?

**Review by ProgramMax — COMMENTED** [review]
post-facto LGTM
